flow_action_components/DedupeRecordCollection/force-app/main/default/classes/DedupeRecordCollection.cls

```diff
@@ -0,0 +1,48 @@
+global with sharing class DedupeRecordCollection {
+    global class DedupeRecordException extends Exception {}
+
+    @InvocableMethod(label='Dedupe Records from Collection' description='Dedupe a list of records using a given field as the unique value and return the unique collection of records' category='Util')
```

Comment:
The two actions are pretty similar so we should give thought to how someone coming at this for the first time would figure out which to use. There's an argument to be made for combining them into one action which would return both kinds of output. But that does require a user who just wants a simple solution to specify an output collection type even if they're not going to use it, which may be confusing.
I think I'd name the actions to focus on their differentiation:
GetUniqueFieldsFromCollection
DedupeRecordCollection

Reply:
I agree @alexed1 , and per my initial comment on the PR, `DedupeFieldsFromCollection` (your proposal as `GetUniqueFieldsFromCollection`) extends functionality of the existing `ExtractFieldsFromCollection` by providing that `dedupeValues` parameter, plus a few extra enhancements. My inclination would be to merge those two and then add the `DedupeRecordCollection` as a new action. 

FWIW--we did start this as a single action, but figured the uses cases for each were mutually exclusive.

Reply:
Agree that at this point the uniqueness chops should probably just be added to ExtractFieldsFromCollection. Please feel free to make that change if you have time.